crates/core/Cargo.toml

```diff
@@ -92,6 +92,9 @@ criterion = "0.5"
 ctor = "0"
 datatest-stable = "0.3"
 deltalake-test = { path = "../test" }
+deltalake-aws = { path = "../aws" }
+deltalake-azure = { path = "../azure" }
+deltalake-gcp = { path = "../gcp" }
```

Comment:
Maybe not a problem but I wonder if it would make sense to have separate tests in each of the object store specific libraries to avoid these dependencies, and have some shared test code in the `deltalake-test` crate.